app/agents/voice/automatic/tools/breeze/analytics.py

```
@@ -245,6 +289,243 @@ async def get_breeze_address_data(params: FunctionCallParams):
+            result = response.json()
+            logger.info(f"Created surcharge rules response status: {response.status_code}, data: {json.dumps(result)}")
+            
+            await params.result_callback({
+                "success": True,
+                "data": result,
+                "message": f"Successfully created {len(rules)} surcharge rules for {payment_type}"
+            })
+    except httpx.HTTPStatusError as e:
+        logger.error(f"HTTP error creating surcharge rules: {e.response.status_code} - {e.response.text}")
+        await params.result_callback({"success": False, "error": f"API error: {e.response.status_code}", "details": e.response.text})
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)